charts/nginx-ingress/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: nginx-ingress
-version: 1.3.0
+version: 1.4.0
```

```suggestion
version: 1.3.0
```
These are automatically updated during major release